osh/cmd_parse.py

```diff
@@ -1399,6 +1399,23 @@ def ParseFor(self):
             n2.redirects = self._ParseRedirectList()
             return n2
 
+    def _ParseConditionList(self):
+        # type: () -> condition_t
+        """
+        condition_list: command_list
+        Looking at `command_list`
```

Comment:
I think this "looking at" comment should name a `self.cur_word` if it exists

Reply:
Not quite sure what I should put here then. The `command_list` could start with a variety of different words.

Should we just not have a "looking at" here?

Reply:
Sure you can remove it if it's not helpful